docker-compose.yml

```diff
@@ -81,16 +81,29 @@ services:
       - '8084:8084'
 
   keycloak-service:
-    image: quay.io/keycloak/keycloak:24.0.3
+    image: quay.io/phasetwo/phasetwo-keycloak:24.0.3
     ports:
       - '8888:8080'
     environment:
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
+      KC_DB: postgres
+      KC_DB_URL: jdbc:postgresql://postgres/keycloak
+      KC_DB_USERNAME: keycloak
+      KC_DB_PASSWORD: keycloak
     volumes:
       - ./keycloak/imports:/opt/keycloak/data/import
     command: -v start-dev --import-realm
 
+  postgres:
```

Comment:
I added this because in order to test that feature you need to manually modify the keycloak db. This can be removed if we decide that for development we don't need keycloak fully setup.